.github/workflows/snl-ci.yml

```
@@ -53,3 +53,39 @@ jobs:
         working-directory: kokkos/build
         run: ctest --output-on-failure --timeout 3600
 
+  HIP_5_6_1:
+    name: SNL_HIP_ROCM_5_6_1
+    runs-on: [snl-kk-env-openblas-0.3.23-hip-5.6.1-latest]
+
+    steps:
+      - name: checkout_kokkos
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: kokkos/kokkos
+          path: kokkos
+
+      - name: configure_kokkos
+        run: |
+          rocm-smi
+          cd kokkos
+          cmake -S . -B build \
+            -DCMAKE_CXX_STANDARD=20 \
+            -DKokkos_ENABLE_HIP=ON \
+            -DKokkos_ARCH_AMD_GFX90A=ON \
+            -DKokkos_ARCH_NATIVE=ON \
+            -DCMAKE_CXX_EXTENSIONS=OFF \
+            -DBUILD_SHARED_LIBS=OFF \
+            -DKokkos_ENABLE_DEPRECATED_CODE_4=OFF \
+            -DKokkos_ENABLE_TESTS=ON \
+            -DKokkos_ENABLE_EXAMPLES=ON \
+            -DKokkos_ENABLE_BENCHMARKS=ON
+
+      - name: build_and_install_kokkos
+        working-directory: kokkos
+        run: |
+          cmake --build build $(nproc)
```

where does `nproc` come from? Also, should it be prefixed by `-j`?
```suggestion
          cmake --build build -j$(nproc)
```

---

`nproc` is a utility that prints the number of CPUs. Usually it's installed, I don't know off the top of my head weather it is in this one. We will find out.